.pre-commit-config.yaml

```diff
@@ -1,23 +1,23 @@
 repos:
   - repo: https://gitlab.com/pycqa/flake8
-    rev: 3.7.9
+    rev: 3.8.3
```

Comment:
should this be included in the commit?

Reply:
I think so, otherwise the local pre-commit is outdated vs the one from CI.

Reply:
I don't quite follow why these changes are included in this pr?

Reply:
@sarah-clements the new repo versions I put in the pre-commit config are the ones currently used on the Travis CI. Without them, there's a linting mismatch.

Either they are accepted here or the ones from Travis CI are downgraded.

Reply:
ok